docs/sql-guide/system-tables/system-tables-home.md

```diff
@@ -38,4 +38,4 @@ The fb_table_ddl table provides the corresponding Data Definition Language (DDL)
 
 ## fb_performance_counters
 
-The fb_performance_counters table provides metrics about the nodes/workers within the database.
\ No newline at end of file
+The fb_performance_counters table provides metrics about the nodes/workers within the database.
```

Comment:
Question : does this relate to community clusters?

Reply:
yes but this isn't  even a change. I'm removing this from the PR to avoid merge conflicts with other PRs